client/src/data/constants.tsx

```diff
@@ -4,21 +4,28 @@ import {defineMessages} from 'react-intl';
 
 // URLS
 export const DOWNLOAD_ZIP_URL = 'https://justice40-data.s3.amazonaws.com/data-pipeline/data/score/downloadable/Screening+Tool+Data.zip';
-export const FEATURE_TILE_BASE_URL = 'https://d2zjid6n5ja2pt.cloudfront.net';
+
+// This is the URL for the CDN that delivers the tile data
+export const FEATURE_TILE_BASE_URL = 'https://d3jqyw10j8e7p9.cloudfront.net';
```

Comment:
Can you verify this url doesn't change regularly? (e.g. `d2zjid6n5ja2pt.cloudfront.net` became `d3jqyw10j8e7p9.cloudfront.net`).

Should this be in a config file or somewhere it can be updated without pushing code/triggering a rebuild?

Reply:
yes these don't change. It was purposefully designed to separate the app CDN from the data CDN. CDN URLs are static.

yes it should be a config! It is a ticket to get these into env variables